api/src/utils/filtersFactorySlots.ts

```diff
@@ -0,0 +1,24 @@
+import type Filters from "@src/types/Filters";
```

Comment:
Do you need filters for these ?

Reply:
That depends on what we decide to do on the previous matter (GetAllSlots).

Reply:
:+1: